src/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "VPN Recommendation Shield Study (bug 1478770)",
+  "description": "This study extension recommends a VPN software solution to those users who it deems likely to benefit from it.",
```

Comment:
Please go over the name and description with a content strategist, this is usually important to not freak out users.